### DISCUSSION TITLE
亲爱的大佬，电视剧第二季还没显示集数的电视剧订阅不了

### DISCUSSION THREAD

**Discussion by ainicc** [jxxghp/MoviePilot#3460] [讨论]
### 当前程序版本

v1.8.0

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

我想订阅 地狱公使 第二季 和 僵尸校园 第二季 和 鱿鱼游戏 第二季 但TMBD还没有集数订阅不了，是我订阅的方式不对吗？

![photo_2024-04-12_16-31-51](https://github.com/jxxghp/MoviePilot/assets/107430324/ca6b31a1-053b-4ba7-b49e-6d6ec6bdfd8d)

![photo_2024-04-12_16-31-56](https://github.com/jxxghp/MoviePilot/assets/107430324/5382989e-88eb-4c2f-bb5e-e5f9d5c2c1e6)

**Comment by madswaord** [comment]
去tmdb里面添加集数就行了
